Consider pin pytest-lazy-fixture version

Hello,

During the package of the new verison 0.26.0  on Debian [0] I have this error:

```
....
    calls = normalize_call(callspec, metafunc, valtype, used_keys)
/usr/lib/python3/dist-packages/pytest_lazyfixture.py:112: in normalize_call
    newmetafunc = copy_metafunc(metafunc)
/usr/lib/python3/dist-packages/pytest_lazyfixture.py:86: in copy_metafunc
    copied._ids = copy.copy(metafunc._ids)
E   AttributeError: 'Metafunc' object has no attribute '_ids'

```
This is for the pytest-lazy-fixture version [1]  that is old in Debian [2]
 but this can be detected early if  marshmallow-sqlalchemy pin the minimum
version in setup.py. Please, consider add it. :)

[0] https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=989270
[1] https://github.com/TvoroG/pytest-lazy-fixture/issues/44
[2] https://tracker.debian.org/pkg/python-pytest-lazy-fixture

--- a/setup.py
+++ b/setup.py
@@ -4,7 +4,7 @@ from setuptools import setup, find_packages
 
 INSTALL_REQUIRES = ("marshmallow>=3.0.0", "SQLAlchemy>=1.2.0")
 EXTRAS_REQUIRE = {
-    "tests": ["pytest", "pytest-lazy-fixture"],
+    "tests": ["pytest", "pytest-lazy-fixture>=0.6.2"],
     "lint": ["flake8==3.9.2", "flake8-bugbear==21.4.3", "pre-commit~=2.0"],
     "docs": ["sphinx==4.1.1", "alabaster==0.7.12", "sphinx-issues==1.2.0"],
 }

